src/tests/class_tests/openms/source/SpectrumExtractor_test.cpp

```diff
@@ -56,224 +55,225 @@ spectrum.resize(73);
 MSSpectrum::Iterator it = spectrum.begin();
 
 it->setMZ(61.92);
```

Comment:
Be sure to put a small explanation on where the data is coming from.  I believe I mentioned previously where the chromatogram came from but if not, I can come up with something for the next PR.

Reply:
Currently there's an explanation on lines 47-51:
```
/////////////////////////////////////////////////////////////
// Raw spectrum data acquired in DDA mode (i.e., product ion full spectrum scan)
// measured on a QTRAP 5500 corresponding to C-Aconitate
// taken from E. coli grown on glucose M9 during steady-state
// for flux analysis.
```

Reply:
I have no idea how I missed that...my apologies.